clb/src/Clb/EmulatedLedgerState.hs

```diff
@@ -59,6 +59,7 @@ initialState params tc =
                 , L.ledgerIx = minBound
                 , L.ledgerPp = params
                 , L.ledgerAccount = L.AccountState (L.Coin 0) (L.Coin 0)
+                , L.ledgerMempool = False
```

Comment:
This is a new field, and I personally don't know it's purpose and for now have just put `False` here.

Reply:
Fwiw I ran bet-ref tests, and they succeeded.